packages/engine/src/avatar/AnimationSystem.ts

```diff
@@ -33,6 +44,36 @@ export default async function AnimationSystem(world: World) {
   return () => {
     const { delta } = world
 
+    for (const entity of desiredTransformQuery(world)) {
```

Comment:
i like this change, maybe we can rename DesiredTransformComponent to something more descriptive, something like AnimateLerpToTransformQuery maybe?